### DISCUSSION TITLE
CAN'T GENERATE DIFFERENT SIZE PAPER LIKE A5 OR B5 OTHER THAN A4. DOMPDF ISN'T WORKING

### DISCUSSION THREAD

**Discussion by ARIA-Mydoc** [dompdf/dompdf#3366] [General]
THE DOCUMENT TEXT DOES NOT DISPLAY ON THE PAPER WITHIN PAGE SIZE DIMENSIONS. I HAVE TRIED DIFFERENT METHODS:
- I FOLLOWED YOUR PAGE SIZE INSTRUCTIONS AND DIDN'T WORK HERE: https://github.com/dompdf/dompdf/blob/v0.8.2/src/Adapter/CPDF.php#L45
- I APPLIED MARGINS ON MY DOCUMENT YET THE SAME THING DIDN'T WORK
- EVEN WHEN I CLICK PRINT > PROPERTIES > CHOOSE LANDSCAPE & A5 SIZE PAPER, IT DOES NOT WORK
- I FOLLOWED THE INSTRUCTIONS AND TROUBLESHOOTINGS OF OTHER PEOPLE POSTS ON SOF BUT STILL HAVING SAME ISSUE: https://stackoverflow.com/questions/25376901/how-to-set-custom-width-and-height-of-pdf-using-dompdf

HOPELESS NOW. IS IT A LIBRARY PROBLEM? OR WHAT IS THE MATTER?

**SEE ATTACHEDD SCREENSHOTS**
![SuperScreenshot 2024-1-4 23-24-4](https://github.com/dompdf/dompdf/assets/73610336/19698bf7-80c6-46fc-a432-97f2fe0d7fa7)
![SuperScreenshot 2024-1-4 23-27-10](https://github.com/dompdf/dompdf/assets/73610336/ebfe7bea-2479-40e8-a70b-3c45fc034e67)
![SuperScreenshot 2024-1-4 23-31-45](https://github.com/dompdf/dompdf/assets/73610336/5754275e-b8b9-4fa4-8b2f-ec2b454e0912)
![SuperScreenshot 2024-1-4 23-35-48](https://github.com/dompdf/dompdf/assets/73610336/90ec627c-57ad-4408-85a9-d3a6e1822245)

**Comment by bsweeney** [comment]
From your description it's hard to tell if the issue is that the HTML layout is not adhering to a5 dimensions or if the issue is that the PDF is using dimensions other than what you specify. Can you supply a copy of the HTML and the generated PDF?

**Reply by ARIA-Mydoc** [reply]
Here's the PDF sample attached
[invoice1353772.pdf](https://github.com/dompdf/dompdf/files/13840932/invoice1353772.pdf)
[mc_112.pdf](https://github.com/dompdf/dompdf/files/13840930/mc_112.pdf)

**Reply by bsweeney** [reply]
And how are you setting the page size? And which version of Dompdf are you using?
